src/index.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable import/extensions */
```

Comment:
- [ ]  Linters are used to analyze the static source code of your software project to look for problems and flag them. This is the reason it is not good to disable linters. Kindly enable the linter and fix the problem.

Reply:
Hello @elisha2kyakpo1 thanks for the review I correct the .eslintrc.json file as I checked and realised it was not up to date with the one here https://github.com/microverseinc/linters-config/blob/master/html-css-js/.eslintrc.json